src/frontend/src/flows/recovery/recoverWith/phrase.ts

```diff
@@ -312,36 +303,40 @@ export const wordTemplate = ({
   </li>`;
 };
 
-type TemplateProps<
-  T extends { tag: "ok" },
-  E extends { tag: "err"; message: string | DynamicKey }
-> = Parameters<typeof recoverWithPhraseTemplate<T, E>>[0];
+type TemplateProps<T> = Parameters<typeof recoverWithPhraseTemplate<T>>[0];
 
-export const recoverWithPhrasePage = <
-  T extends { tag: "ok" },
-  E extends { tag: "err"; message: string | DynamicKey }
->(
-  props: TemplateProps<T, E>,
+export const recoverWithPhrasePage = <T>(
+  props: TemplateProps<T>,
   container?: HTMLElement
-) => renderPage(recoverWithPhraseTemplate<T, E>)(props, container);
+) => renderPage(recoverWithPhraseTemplate<T>)(props, container);
 
 export const recoverWithPhrase = ({
   connection,
   message,
 }: {
   connection: Connection;
   message: TemplateResult | string;
-}): Promise<LoginFlowSuccess | LoginFlowCanceled> => {
+}): Promise<LoginSuccess | { tag: "canceled" }> => {
   return new Promise((resolve) => {
-    recoverWithPhrasePage<LoginFlowSuccess, LoginFlowError>({
+    recoverWithPhrasePage<LoginSuccess>({
       confirm: (result) => resolve(result),
       verify: async ({ userNumber, words }) => {
-        const result = await withLoader(async () =>
-          apiResultToLoginFlowResult(
-            await connection.fromSeedPhrase(userNumber, words.join(" "))
-          )
+        const result = await withLoader(() =>
+          connection.fromSeedPhrase(userNumber, words.join(" "))
         );
-        return result;
+        if (result.kind !== "loginSuccess") {
+          const msg = {
+            noSeedPhrase:
+              "No recovery phrase: there is no recovery phrase associated with this identity. Cancel and recover another way.",
+            seedPhraseFail:
+              "Invalid Seed Phrase: Failed to authenticate using this seed phrase. Did you enter it correctly?",
```

Comment:
Taken from `flowResult.ts` where they could have actually never been triggered (there errors were not allowed to bubble up) and added here